analytics/eoy_2016.js

```diff
@@ -0,0 +1,18 @@
+(function () {
+  'use strict'
+
+  // DOM events
+  var $document = $(document)
+
+  $document.on('click', '.eoy-artist-feature .avant-garde-button-white', function (e) {
+    analytics.track('Clicked EOY promotion read article', {
+      destination_path: $(this).attr('href'),
```

Comment:
Yeah, not sure. I'm taking directly whats in the `article_link` field in  https://www.artsy.net/eoy_2016/data